src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
> Aren't there other cases where some of the raw P/Invokes have been left as internal rather than private?

Yes, but they either don't need normalization or haven't gotten it yet (which was my comment on wrapping to avoid needing to explicitly specify the StringBuilder size outside of interop).

>at that point the need for these  Interop.*.cs  files would go away and we'd be left with the PathInternal files that would include this behavior

I could simply remove these from interop and put them directly in the PathInternal. That does, however, risk reintroduction and missing the important wrapping behavior. I suppose we could keep empty stub files that say "use PathInternal".

I have a hard time seeing the value beyond aesthetics. The way I've got it doesn't look as clean, but it has no chance of accidental misuse. I suppose I can buy that it makes it more difficult to enforce restricting the interop layer to normalizing access to the Windows APIs (because it is harder to audit). As such I'd be fine with stubbing the files and pushing the P/Invokes (in this case) to PathInternal.